docs/vertx-runtime/master.adoc

```diff
@@ -44,12 +44,16 @@ include::topics/assembly_building-and-deploying-the-booster-to-openshiftcontaine
 
 include::topics/rest-level-0-mission-basic-interaction-vertx.adoc[leveloffset=+3]
 
+include::topics/proc_running-integration-tests.adoc[leveloffset=+3]
+
 include::topics/rest-level-0-mission-resources.adoc[leveloffset=+3]
 
 
 [[mission-configmap-vertx]]
 === {mission-configmap-vertx-guide-name}
 :configmap-vertx:
+:parameter-runtime: vertx
+:parameter-mission-name:
```

Comment:
Question: What does this do, exactly?

Reply:
My current assumption is that this is a leftover form an earlier version of the change, will investigate and do a cleanup

Reply:
Upon investigation, I have discovered that these parameters are unused. Will remove in the next pass.

Reply:
EDIT: found a use for both of those parameters:
* `parameter-runtime` can be used in the module anchor and in the `context` attrrbute
* `parameter-mission-name` can be used to create a customized title when combined with `parameter-runtime-name` 

I'll try to incorporate that.